docs/content/release-process.md

```diff
@@ -9,16 +9,16 @@ url = "https://github.com/fermyon/spin/blob/main/docs/content/release-process.md
 To cut a release of Spin, you will need to do the following:
 
 1. Create a pull request that changes the version number for your new version
-   (e.g. `0.7.0` becomes `0.7.1`)
+   (e.g. `0.7.1` becomes `0.7.2`)
```

Comment:
We don't need to update the example numbers each time do we?!

Reply:
No, I suppose not :).  I can remove the changes to this file if we'd like, so we don't get into the habit/expectation.